readme.md

```diff
@@ -93,6 +93,13 @@ With plain `fetch`, it would be:
 })();
 ```
 
+In environments that do not support `import`, you can load `ky` in UMD format. For example, using `require()`:
```

Comment:
Can you linkify `UMD format` to somewhere that describes what it is?

Reply:
Surprisingly, the UMD repository on GitHub does a pretty poor job of explaining what it is and its purpose. I couldn't quickly find anything succinct. I did find a good [article on module systems](https://medium.freecodecamp.org/anatomy-of-js-module-systems-and-building-libraries-fadcd8dbd0e) that covers UMD. I'll link that, but happy to update it if anyone finds something better.